raiden/network/transport/matrix/utils.py

```diff
@@ -270,6 +297,18 @@ def _fetch_user_presence(self, user_id: str) -> UserPresence:
     def _validate_userid_signature(user: User) -> Optional[Address]:
         return validate_userid_signature(user)
 
+    @property
+    def log(self) -> BoundLoggerLazyProxy:
```

Comment:
This only seems to run once, why can't you do that in the constructor or at the end of `start()`? Like this it's unclear when this is first run.

Reply:
Actually this is a bit annoying. The log can only be bound once the client is logged in (because only then the user id is available) however the AddressManager must be started before that so it can catch events immediately after the login happens. :/

I've slightly improved it now so there is at least no exception if a logging call happens to be made before login and also the binding is undone on `.stop()`.